packages/app/pubspec.lock

```diff
@@ -148,7 +148,7 @@ packages:
   dynamite_runtime:
     dependency: transitive
     description:
-      path: "../dynamite/dynamite_runtime"
+      path: "../../../../../../.pub-cache/git/nextcloud-neon-482bff006d4338964c0704600c50434fe76a19b2/packages/dynamite/dynamite_runtime"
```

Comment:
that's odd as it references my home folder :/

Probably a pub bug.

Reply:
Maybe we need to also add an override for dynamite_runtime to app.

This probably references my home dir as the git dependency gets cached there. A perfect example of the above mentioned problem.

Reply:
Looks like the CI agrees with me

Reply:
Yeah the other packages also need the override, otherwise they won't use the local version.

Reply:
That's odd. I just added an override and both `flutter analyze` and `flutter pub get` work in the `packages/app` dir and they work fine.